assets/pages/new-contributions-landing/components/SetPassword/setPassword.scss

```diff
@@ -0,0 +1,48 @@
+.set-password {
+  .form__input--email {
+    pointer-events:none;
+    background-color: #dcdcdc;
+
+  }
+  
+  .form__field--contribution-password {
+    margin-top: $gu-v-spacing;
+  }
+
+  .form__submit-button--create-account {
+    width: 100%;
+    position: relative;
+  }
+
+  .svg-arrow-right-straight {
+    position: absolute;
+    right: $gu-v-spacing;
+  }
+
+  .form__submit {
+    padding-bottom: 0;
+  }
+
+  .component-cta-link--no-thanks {
+    background-color: transparent;
+    margin-top: $gu-v-spacing;
+    height: 42px;
+    color: gu-colour(garnett-neutral-1);
+    border: 1px solid gu-colour(neutral-1);
+    display: inline-block;
+    position: relative;
+    .svg-arrow-right-straight {
+      fill: gu-colour(garnett-neutral-1);
+      position: absolute;
+      right: $gu-v-spacing;
+      width: 20px;
+      height: 18px;
+      vertical-align: middle;
```

Comment:
Pretty sure this `vertical-align: middle` isn't doing anything if this element is absolutely positioned.

I'd argue for flex-box being our new standard method of centering things, e.g. here (styles applied to the parent element): https://github.com/guardian/support-frontend/pull/972/commits/3de29c7c8e4bf08404644c2edf7d3af4d4843e73

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container

Reply:
... or going back on that, simpler methods especially if already used for other similar elements (see comment above)

unless I'm mistaken, in this case it's just an identical button with a different background colour so I feel like we could get rid of quite a lot of this CSS?